packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1174,6 +1197,33 @@ class EditableText extends StatefulWidget {
   ///    and notifies its listeners on [TextEditingValue] changes.
   final ValueChanged<String>? onChanged;
 
+  /// {@template flutter.widgets.editableText.onContentInserted}
+  /// Called when a user inserts content through the virtual / on-screen keyboard,
+  /// currently only used on Android.
+  ///
+  /// The following data is returned:
```

Comment:
nit: maybe move this to `KeyboardInsertedContent` and just say that the `KeyboardInsertedContent` contains the data?